transports/transports.go

```diff
@@ -2,52 +2,24 @@ package transports
 
 import (
 	"fmt"
-	"strings"
 
-	"github.com/containers/image/directory"
-	"github.com/containers/image/docker"
-	"github.com/containers/image/docker/daemon"
-	ociLayout "github.com/containers/image/oci/layout"
-	"github.com/containers/image/openshift"
-	"github.com/containers/image/storage"
 	"github.com/containers/image/types"
-	"github.com/pkg/errors"
 )
 
 // KnownTransports is a registry of known ImageTransport instances.
 var KnownTransports map[string]types.ImageTransport
```

Comment:
IMO this should be hidden (with some sort of `Get` function) and have a lock around it internally. It isn't a good idea to expose a `map` globally.